content/general/concepts/nft/nft.md

```diff
@@ -0,0 +1,30 @@
+---
+Title: 'NFT'
+Description: 'A non-fungible token (NFT) acts as a digital representation to prove ownership for a digital good backed by blockchain technology.'
+Subjects:
+  - 'Computer Science'
+  - 'Web Development'
+Tags:
+  - 'Blockchain'
+  - 'Web3'
+  - 'Cryptocurrency'
+CatalogContent:
+  - 'introduction-to-blockchain'
+  - 'paths/computer-science'
+---
+
+A non-fungible token (NFT) acts as a digital representation to prove a ownership for a digital good backed by blockchain technology.
+
+The term "token" in the phrase "non-fungible token" refers to this digital representation. "Non-fungible" simply means "non-replaceable". It can't be replaced, there is only one of them. It is always unique.
+
+For example, a 1 GBP note is fungible because you are able to replace that note with another note. However, each NFT is always unique, and you cannot simply swap for another different NFT, even if they are both a representation of the same good. This means that non-fungible things can be far more valuable than fungible things.
```

Comment:
```suggestion
For example, a 1 U.S. Dollar note is fungible because you are able to replace that note with another note. However, each NFT is always unique, and you cannot simply swap for another different NFT, even if they are both a representation of the same good. This means that non-fungible things can be far more valuable than fungible things.
```